js/main.js

```diff
@@ -0,0 +1,26 @@
+
+var $emoji = document.querySelector('.emoji');
+$emoji.addEventListener('click', emojiHandler);
+
+var showOrHide = true;
+
+function emojiHandler(event) {
+  if (showOrHide === true) {
```

Comment:
Hmmm, `showOrHide` shouldn't be necessary, you can probably remove that variable and these conditionals, and just keep the work in the conditionals' code blocks